c10/cuda/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ configure_file(
 # and headers you add
 set(C10_CUDA_SRCS
     CUDAStream.cpp
+    CUDAFunctions.cpp
```

Comment:
See note about about adding new source file to hipify mappings.

Reply:
https://github.com/pytorch/pytorch/blob/cc906ee6190699840048497e575aa27f2c5c63e2/c10/cuda/CMakeLists.txt#L19-L24

Reply:
I've seen that, but I don't see any .cpp files listed in cuda_to_hip_mappings.py. Note that I'm adding a .cpp file for an existing CUDAFunctions.h file and it's already listed in cuda_to_hip_mappings.

Also there's some special casing for device_count() in it but I can't figure out how it works: https://github.com/pytorch/pytorch/blob/master/torch/utils/hipify/cuda_to_hip_mappings.py#L8069

Reply:
Seems like it was the same missing C10_CUDA_API macro problem